storybookjs/storybook
# UI: Add aria-labels to buttons without discernible text

Issue: https://github.com/storybookjs/storybook/issues/13797

## What I did
Added aria-label attributes with values to expandable text and toggle indicator
## How to test
Run axe devtools and observe that the error indicated in the issue above are no longer being flagged

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
